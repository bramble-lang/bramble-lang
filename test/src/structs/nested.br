fn my_main() {
    let t:MyStruct2 := MyStruct2{
        a: 123,
        ms: MyStruct{x: 1, z: 3, y: 2},
        b: 456,
    };

    printiln t.a;
    printiln t.ms.x;

    let t2:MyStruct := MyStruct {
        x: 10,
        y: 11,
        z: 12,
    };
    printiln t2.x;
    
    return;
}

struct MyStruct {
    x: i32,
    y: i32,
    z: i32,
}

struct MyStruct2 {
    a: i32,
    ms: MyStruct,
    b: i32,
}