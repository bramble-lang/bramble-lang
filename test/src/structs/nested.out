123
1
2
3
456
666
123
1
2
3
456
true
